recommended_ruleset.js

```diff
@@ -185,6 +186,7 @@ module.exports = {
          * Accessibility. The following rules should be turned on to guarantee the best user
          * experience for keyboard and screen reader users.
          */
+        'react-a11y-accessible-headings': false,
```

Comment:
Shouldn't this one be `true`? I don't see `recommendation: false` in rule file metadata.

Sorry. Previously added comment to wrong line. `ban-ts-ignore` is expected to be `false`. At leas ATM.

Reply:
I mean that this rule is in this repo (not in `tslint`) and next person that will run `npm run generate:recommendations` will see diff in this line unless `recommendation: 'false,'` will be added to [rule metadata](https://github.com/Microsoft/tslint-microsoft-contrib/blob/b2fa5c7a8b863ebbd53b341081a7576980230f21/src/reactA11yAccessibleHeadingsRule.ts#L16-L31).